session6/src/player.py

```diff
@@ -0,0 +1,72 @@
+import pygame
+import os
+
+
+GRAVITY = 0.25
+JUMP_VELOCITY = -1 * 12
+
+
+class Player(pygame.sprite.Sprite):
+    def __init__(self):
+        super().__init__()
+        self.x_speed = 5  # speed in the x direction
```

Comment:
Any reason why this is initialized to 5 not 0?
```suggestion
        self.x_speed = 0  # Speed in the x direction
```

Reply:
You won't be able to move at all if it's set to zero 😉 x acceleration wasn't covered